source/extensions/filters/http/fault/fault_filter.cc

```diff
@@ -75,6 +75,7 @@ Http::FilterHeadersStatus FaultFilter::decodeHeaders(Http::HeaderMap& headers, b
   // faults. In other words, runtime is supported only when faults are
   // configured at the filter level.
   fault_settings_ = config_->settings();
+
```

Comment:
revert unrelated change

Reply:
Sorry, forgot to check before submission